test/python/transpiler/test_permrowcol.py

```diff
@@ -493,35 +493,45 @@ def test_reduce_graph_removes_edges_from_graph(self):
         permrowcol._reduce_graph(2)
         self.assertCountEqual(permrowcol._graph.edge_list(), [])
 
-    def test_perm_row_col_does_correct_permutation_matrix(self):
-        """Test Not to be included to the final commit"""
-        coupling_list = [(0, 1), (0, 3), (1, 2), (1, 4), (2, 5), (3, 4), (4, 5)]
+    def test_no_hadamard_gates_added_with_complete_graph(self):
+        """Test that no hadamard gates are added if graph has bidirectional edges"""
+        n = 6
+        parity_mat = build_random_parity_matrix(42, n, 60)
+        coupling_list = [(i, j) for i in range(n) for j in range(n) if i != j]
         coupling = CouplingMap(coupling_list)
         permrowcol = PermRowCol(coupling)
-        parity_mat = np.array(
-            [
-                [0, 1, 0, 1, 1, 0],
-                [1, 1, 1, 1, 1, 0],
-                [1, 0, 0, 0, 1, 1],
-                [1, 1, 1, 0, 1, 0],
-                [1, 0, 1, 0, 1, 0],
-                [1, 0, 1, 0, 1, 1],
-            ]
-        )
+        circuit, perm = permrowcol.perm_row_col(parity_mat)
+        h_gates = []
+        for i in circuit:
+            if i[0].name == "h":
+                h_gates.append(i)
+        self.assertEqual(len(h_gates), 0)
 
-        correct_permutation_matrix = np.array(
-            [
-                [0, 0, 0, 1, 0, 0],
-                [0, 0, 1, 0, 0, 0],
-                [0, 0, 0, 0, 0, 1],
-                [0, 1, 0, 0, 0, 0],
-                [0, 0, 0, 0, 1, 0],
-                [1, 0, 0, 0, 0, 0],
-            ]
-        )
+    def test_add_cnot_adds_four_hadamard_gates_if_cnot_is_in_wrong_direction(self):
+        """Test add one cnot to wrong direction adds four cnots"""
```

Comment:
The doc string is incorrect as it says that four cnots are added instead of four hadamard gates